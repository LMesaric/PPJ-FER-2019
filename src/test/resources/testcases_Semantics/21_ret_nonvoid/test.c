void main(void) {
    return;
    return 1;
    return 'a';
    return 3 + 5;
}
