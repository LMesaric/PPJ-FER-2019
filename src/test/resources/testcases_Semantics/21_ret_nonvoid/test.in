<prijevodna_jedinica>
 <vanjska_deklaracija>
  <definicija_funkcije>
   <ime_tipa>
    <specifikator_tipa>
     KR_VOID 1 void
   IDN 1 main
   L_ZAGRADA 1 (
   KR_VOID 1 void
   D_ZAGRADA 1 )
   <slozena_naredba>
    L_VIT_ZAGRADA 1 {
    <lista_naredbi>
     <lista_naredbi>
      <lista_naredbi>
       <lista_naredbi>
        <naredba>
         <naredba_skoka>
          KR_RETURN 2 return
          TOCKAZAREZ 2 ;
       <naredba>
        <naredba_skoka>
         KR_RETURN 3 return
         <izraz>
          <izraz_pridruzivanja>
           <log_ili_izraz>
            <log_i_izraz>
             <bin_ili_izraz>
              <bin_xili_izraz>
               <bin_i_izraz>
                <jednakosni_izraz>
                 <odnosni_izraz>
                  <aditivni_izraz>
                   <multiplikativni_izraz>
                    <cast_izraz>
                     <unarni_izraz>
                      <postfiks_izraz>
                       <primarni_izraz>
                        BROJ 3 1
         TOCKAZAREZ 3 ;
      <naredba>
       <naredba_skoka>
        KR_RETURN 4 return
        <izraz>
         <izraz_pridruzivanja>
          <log_ili_izraz>
           <log_i_izraz>
            <bin_ili_izraz>
             <bin_xili_izraz>
              <bin_i_izraz>
               <jednakosni_izraz>
                <odnosni_izraz>
                 <aditivni_izraz>
                  <multiplikativni_izraz>
                   <cast_izraz>
                    <unarni_izraz>
                     <postfiks_izraz>
                      <primarni_izraz>
                       ZNAK 4 'a'
        TOCKAZAREZ 4 ;
     <naredba>
      <naredba_skoka>
       KR_RETURN 5 return
       <izraz>
        <izraz_pridruzivanja>
         <log_ili_izraz>
          <log_i_izraz>
           <bin_ili_izraz>
            <bin_xili_izraz>
             <bin_i_izraz>
              <jednakosni_izraz>
               <odnosni_izraz>
                <aditivni_izraz>
                 <aditivni_izraz>
                  <multiplikativni_izraz>
                   <cast_izraz>
                    <unarni_izraz>
                     <postfiks_izraz>
                      <primarni_izraz>
                       BROJ 5 3
                 PLUS 5 +
                 <multiplikativni_izraz>
                  <cast_izraz>
                   <unarni_izraz>
                    <postfiks_izraz>
                     <primarni_izraz>
                      BROJ 5 5
       TOCKAZAREZ 5 ;
    D_VIT_ZAGRADA 6 }
