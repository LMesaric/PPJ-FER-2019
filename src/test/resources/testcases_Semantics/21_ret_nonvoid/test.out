<naredba_skoka> ::= KR_RETURN(3,return) <izraz> TOCKAZAREZ(3,;)
